package.json

```diff
@@ -91,14 +93,15 @@
     "source-map-support": "^0.5.6",
     "sprintf-js": "git+https://github.com/alexei/sprintf.js.git#36b04b33975e597b03737c3ed0a2a49ec1b2e8c4",
     "standard": "^11.0.1",
-    "three": "^0.111.0",
     "ts-jest": "^25.2.1",
-    "typedoc": "^0.16.10",
-    "typescript": "^3.7.3",
+    "typedoc": "^0.16.11",
+    "typescript": "^3.8.3",
     "uglify-js": "^3.4.6"
   },
   "dependencies": {
     "@types/chroma-js": "^1.3.5",
-    "@types/signals": "1.0.1"
+    "@types/signals": "1.0.1",
```

Comment:
Sorry, I keep meaning to come and look at this properly but am not quite sure - are you saying we should move these ones (chroma-js etc) under dependencies as we've done with three.js and let them be supplied by the project that makes use of them?

Reply:
Yes, exactly. When built as a module, the module does not include a copy of chroma-js (for example) -- as indeed it shouldn't. So it has to be listed as a dependency rather than a dev-dependency.

Reply:
Hi Fred -- I'd love to get this merged in any time. We can move the dependencies around in a separate PR if that's helpful.

Reply:
Sorry - I wasn't sure which of chroma-js etc were accessible via the NGL namespace, and I *was* seeing them bundled into the module (whereas you weren't) - I never quite got to the bottom of it but let's merge as it is now.